users/users/users.php

```diff
@@ -40,11 +43,92 @@ public function delete()
 		$this->plugin->setResponse( JText::_( 'PLG_API_USERS_IN_DELETE_FUNCTION_MESSAGE' ));
 	}
 
-	/**
-	 * Function post for create user record.
+	 /**
+	 * Function for edit user record.
 	 *
 	 * @return void
 	 */
+	public function put()
+	{
+		$app              = JFactory::getApplication();
+		$db 			  = JFactory::getDbo();
+		$data             = array();
+		$eobj             = new stdClass();	
+		$eobj->status     = false;
+		$eobj->id         = 0;
+		
+		// Get values paased by put in the url
+		$data = JRequest::get( 'put');
```

Comment:
Done with the suggested changes, can you please review and revert with the
feedback.

On Wed, Aug 24, 2016 at 3:19 PM, Ashwin Date <notifications@github.com>
wrote:

> In users/users/users.php
> <https://github.com/techjoomla/com_api-plugins/pull/36#discussion_r76026851>
> :
>
> >  	 *
> >  	 * @return void
> >  	 */
> > +	public function put()
> > +	{
> > +		$app              = JFactory::getApplication();
> > +		$db 			  = JFactory::getDbo();
> > +		$data             = array();
> > +		$eobj             = new stdClass();	
> > +		$eobj->status     = false;
> > +		$eobj->id         = 0;
> > +		
> > +		// Get values paased by put in the url
> > +		$data = JRequest::get( 'put');
>
> You should be using JInput, not JRequest.
> Also - is this file passing PHPCS ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/techjoomla/com_api-plugins/pull/36/files/1d56b6d66106c77db606fe48395ef93008823368#r76026851>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AUNb4joxZKFeV9x7l-NeHZ2NqEvjAMbJks5qjBOcgaJpZM4JqyuW>
> .
>



-- 
Best Regards,
Deepali A. Madane